packages/access-client/src/agent.js

```diff
@@ -28,6 +32,59 @@ export { AgentData }
 const HOST = 'https://access.web3.storage'
 const PRINCIPAL = DID.parse('did:web:web3.storage')
 
+/**
+ *
+ * @param {string} email
+ * @returns {Ucanto.Principal<Ucanto.DID<'mailto'>>}
+ */
+function emailToSessionPrincipal(email) {
+  const parts = email.split('@').map((s) => encodeURIComponent(s))
+  return DID.parse(`did:mailto:${parts[1]}:${parts[0]}`)
+}
+
+/**
+ * @param {Ucanto.Signer<Ucanto.DID<'key'>>} space
+ * @param {Ucanto.Principal<Ucanto.DID<'mailto'>>} account
+ * @returns
+ */
+async function createSpaceSaysAccountCanAdminSpace(space, account) {
+  return ucanto.delegate({
+    issuer: space,
+    audience: account,
+    capabilities: [
```

Comment:
I think it's best to delegate `{ with: "ucan:*", can: "*" }` to an account so that account has complete authority over the space not just over the capabilities that we use today.

Reply:
seems like a good idea to me. @travis wdyt?

Reply:
sounds right to me!

Reply:
This function isn't the right place to do `ucan:*`, but I did change this to
* allow `capabilities` to be passed as param
* default them to `can="*"` instead of the specific cans used before

https://github.com/web3-storage/w3protocol/pull/433/commits/1ea805e12b5f9ffcedca6e927a41ff58b7c85f28

(I don't disagree that it prob makes sense to do `{ with: "ucan:*", can: "*" }` in delegation from space->account more generally, but this function is named in a way that makes caller think `with=space`